JetNews/build.gradle

```diff
@@ -38,6 +38,8 @@ subprojects {
     repositories {
         google()
         jcenter()
+
+        maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
```

Comment:
Nit: Add a comment saying this is for the snapshot, so that we can remove when needed

Reply:
Done.